calebwagner/sink-repair
# added main html page in javascript module

I created a module that will display all the html that will be rendered to the browser from javascript.

Steps to check code:
1. git fetch --all
2. git checkout cjw-create-html-page
3. code .
4. check in SinkRepair.js module that Requests.js has been imported
5. check in SinkRepair.js module for invoked Requests function
